__tests__/app.test.js

```diff
@@ -293,3 +317,20 @@ describe("DELETE /api/comments/:comment_id", () => {
       });
   });
 });
+
+describe("GET /api/users", ()=>{
+  test("200: responds with a body of an array of all user objects", ()=>{
+    return request(app)
+    .get("/api/users")
+    .expect(200)
+    .then((response)=>{
+      const {body: {users}} = response
+      console.log(users)
+      users.forEach((user)=>{
+        expect(user).toHaveProperty("username")
+        expect(user).toHaveProperty("name")
+        expect(user).toHaveProperty("avatar_url")
+      })
```

Comment:
You should have a length check in here too, if it's an empty array the forEach will just skip 👍